index.js

```diff
@@ -1,7 +1,7 @@
 Cypress.Commands.add('injectAxe', () => {
   cy.window({ log: false }).then(window => {
     const axe = require('axe-core')
-    window.eval(axe.source)
+    window.axe = axe
```

Comment:
We already have access to the `axe` reference, just assign it to the the app's `window` object so it is available to other `cypress-axe` commands. They refer to `axe` via the `window` object already.